### PR TITLE
Make net nn-190f102a22c3.nnue the default net.

### PR DESCRIPTION
**Pull request by MichaelB7**
**LTC**
https://tests.stockfishchess.org/tests/view/60d09f52b4c17000d679517f
LLR: 2.93 (-2.94,2.94) <0.50,3.50>
Total: 32184 W: 1100 L: 970 D: 30114
Ptnml(0-2): 10, 878, 14193, 994, 17

**STC**
https://tests.stockfishchess.org/tests/view/60d086c02114332881e7368e
LLR: 2.93 (-2.94,2.94) <-0.50,2.50>
Total: 11360 W: 1056 L: 906 D: 9398
Ptnml(0-2): 25, 735, 4026, 853, 41

Trained with pytorch.   https://github.com/glinscott/nnue-pytorch

python train.py i:/bin/all.binpack i:/bin/all.binpack --gpus 1 --threads 4 --num-workers 30 --batch-size 16384 --progress_bar_refresh_rate 300 --smart-fen-skipping --random-fen-skipping 3 --features=HalfKAv2^   --lambda=1.0  --max_epochs=440 --seed %random%%random% --default_root_dir exp/run_17 --resume-from-model ./pt/nn-75980ca503c6.pt (previous  master)

all.binpack equaled 4 parts Wrong_NNUE_2.binpack https://drive.google.com/file/d/1seGNOqcVdvK_vPNq98j-zV3XPE5zWAeq/view?usp=sharing plus two parts of Training_Data.binpack https://drive.google.com/file/d/1RFkQES3DpsiJqsOtUshENtzPfFgUmEff/view?usp=sharing
Each set was concatenated together - making one large Wrong_NNUE 2 binpack and one large Training so the were approximately equal in size. They were then interleaved together. The idea was to give Wrong_NNUE.binpack closer to equal weighting with the Training_Data binpack 

**model.py changes:**
     t = outcome
     p = (score / in_scaling).sigmoid()
     **loss = torch.pow(torch.abs(p - q), 2.6).mean()**

  def configure_optimizers(self):
    # Train with a lower LR on the output layer
    **LR = 8.75e-4
    gamma=0.992**
source of suggestion: https://github.com/glinscott/nnue-pytorch/issues/130 .

Bench: 5113198

### CODE REVIEW

**Comment by vondele** [comment]
Nice.

Can I suggest you to make a PR with the exact change you made to the trainer to the nnue-trainer repository as well?
